app/controllers/mailgun_webhooks_controller.rb

```diff
@@ -83,6 +83,8 @@ def create_incoming_email
 
       end
 
+      IntercomService.create_intercom_message_from_email(contact_record, fw_from_hub: true) if (client.tax_returns.pluck(:status).map(&:to_sym) & TaxReturnStatus::FORWARD_TO_INTERCOM_STATUSES).any?
```

Comment:
We do this client.tax_returns.pluck(:status) thing to check for "if client has tax return in a particular status" in at least a handful of places if not more now.

I'm starting to think it makes sense to have a method thats like client.tax_return_in(status) where status can be one or an array of statuses.

Not necessary right now, but it would ultimately make our code drier and more readable to have a convenience method like that.

Reply:
Yeah definitely! I was searching through to code to see if we already had a method for this bc I feel like we do this so much